feature/bgp/gracefulrestart/ate_tests/bgp_graceful_restart_test/bgp_graceful_restart_test.go

```diff
@@ -398,13 +453,19 @@ func TestTrafficWithGracefulRestartSpeaker(t *testing.T) {
 	t.Run("configureDut", func(t *testing.T) {
 		t.Log("Start DUT interface Config")
 		configureDUT(t, dut)
+		if *deviations.RoutePolicyUnderPeerGroup {
+			configureRoutePolicy(t, dut, "ALLOW", oc.RoutingPolicy_PolicyResultType_ACCEPT_ROUTE)
+		}
 	})
 
 	// Configure BGP+Neighbors on the DUT
 	t.Run("configureBGP", func(t *testing.T) {
 		t.Log("Configure BGP with Graceful Restart option under Global Bgp")
 		dutConfPath := gnmi.OC().NetworkInstance(*deviations.DefaultNetworkInstance).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, "BGP")
 		gnmi.Delete(t, dut, dutConfPath.Config())
```

Comment:
we convinced google team that delete is unnecessary in ISIS cases and replace should do what delete is trying to do + update. Same case here no point for delete + replace here, let's simply remove the delete and see how it goes. We can return the delete if they disagree.

Reply:
Removed delete operation and kept only Replace here